impeller/fixtures/ink_sparkle_web.frag

```diff
@@ -0,0 +1,100 @@
+#version 320 es
```

Comment:
What's the difference between this and `ink_sparkle.frag`?

Reply:
No difference, but with the same input file the impellerc build target will have overlapping inputs. Sort of a hacky way to test that it works...

Reply:
You can specify a `intermediates_subdir` option to `impellerc` to get it to put the outputs in a different subdirectory.

Reply:
Done